frontend/src/app/me/me.component.html

```diff
@@ -0,0 +1 @@
+Tutaj znajdzie się widok użytkownika
```

Comment:
Maybe you can plan also the more detailed layout of this view, e.g. put two ` div `s, one for personal data form and the second for the list of organizations, with proper bootstrap classes - placeholders to just put those two components when they will be ready.